e2e/fixtures/test-registrations-OCW.csv

```diff
@@ -0,0 +1,21 @@
+referenceId,preferredLanguage,paymentAmountMultiplier,maxPayments,firstName,lastName,phoneNumber,fspName,whatsappPhoneNumber,addressStreet,addressHouseNumber,addressHouseNumberAddition,addressPostalCode,addressCity
```

Comment:
Is this the same data as in the `121-services/seed-data`-folder?

Reply:
Yes, for now its same file but e2e test could use different dataset to do the testing. Thats why it copied to here.